CHANGELOG.md

```diff
@@ -1,16 +1,4 @@
-## 0.5.0-beta+1
-
-This code is considered production quality, but depends on angular2:
-^3.0.0-alpha+1. The alpha tag represents the evolving nature of the AngularDart
-api, not code quality (3.0.0-beta+1 is used in production Google apps).
-
- * Fix link in CHANGELOG to old pub package.
-
-## 0.5.0-beta
-
-This code is considered production quality, but depends on angular2:
-^3.0.0-alpha+1. The alpha tag represents the evolving nature of the AngularDart
-api, not code quality (3.0.0-beta+1 is used in production Google apps).
+## 0.5.0
 
```

Comment:
The assumption: most folks just want the non-beta release. We shouldn't make folks looking at "Gold" to have to parse through each beta release.